packages/cassowary/lib/cassowary.dart

```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 
 /// An implementation of the Cassowary constraint solving algorithm in Dart.
+/// 
+/// To use, import `package:cassowary/cassowary.dart`.
```

Comment:
ah, so the import line is correct. It's just that the developer also needs to add lines to their pubspec.yaml file.

I can add this additional instruction.

Alternatively, I can add "this library is not meant to be imported by developers. It will soon be integrated into another flutter library"

@abarth would that be accurate to say?